cli/cmd/dashboard.go

```diff
@@ -94,7 +94,7 @@ func newCmdDashboard() *cobra.Command {
 			portforward, err := k8s.NewPortForward(
 				cmd.Context(),
 				k8sAPI,
-				controlPlaneNamespace,
+				linkerdVizNamespace,
```

Comment:
This command will eventually move to the `linkerd viz` subcommand.  Furthermore, we can dynamically detect the namespace using the linkerd.io/extensions label.  We should use that label (or, we can do that in a future PR and just hardcode it here for now)